content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,474 @@
+mv containerd.service /etc/systemd/system/
+
+systemctl enable --now containerd.service
+```
+-->
+
+Quick test:
+
+```shell
+systemctl status containerd.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u containerd
+```
+
+### Kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+```
+-->
+
+Configure:
+
+{{< note >}}
+This tutorial uses the `10.241.1.0` network. Make sure you don't have any
+conflicts with an existing network. If necessary, use a network that is available to
+you.
+{{< /note >}}
+
+```shell
+mkdir -p /etc/kubernetes
+
+cat << EOF > /etc/kubernetes/kubelet.yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false
+authorization:
+  mode: AlwaysAllow
+enableServer: false
+logging:
+  format: text
+  sanitization: false
+readOnlyPort: 10250
+podCIDR: 10.241.1.0/24
+staticPodPath: /etc/kubernetes/manifests
+EOF
+```
+
+Install:
+
+```shell
+chmod +x kubelet
+mv kubelet /usr/bin/
+
+cat <<EOF > /etc/systemd/system/kubelet.service
+[Unit]
+Description=Kubelet
+
+[Service]
+ExecStart=/usr/bin/kubelet \
+ --config=/etc/kubernetes/kubelet.yaml
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable --now kubelet.service
+```
+
+Quick test:
+
+```shell
+systemctl status kubelet.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u kubelet
+```
+
+## Run a static Pod in the Kubelet
+
+Configure:
+
+```shell
+cat <<EOF > static-web.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+EOF
+```
+
+Deploy:
+
+```shell
+mkdir -p /etc/kubernetes/manifests
+```
+
+Move the `static-web.yaml` manifest file to the `/etc/kubernetes/manifests` directory.
+
+```shell
+mv static-web.yaml /etc/kubernetes/manifests/
+```
+
+### Find out information about the Kubelet and the static Pod
+
+The cni-plugin creates a `veth` interface, retrieve the information:
+
+```shell
+ip link show | grep veth
+```
+
+The output is similar to:
+
+```
+6: veth85d4e050@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default
+```
+
+Use the output to retrieve the IP address of the `veth` interface:
+
+```shell
+ip addr show veth85d4e050 | grep inet
```

Comment:
Could use something like:
```shell
ip -f inet address show dev veth12ab34bc
```

Reply:
Okay but the output is still very verbose, so I'll keep the `| grep inet`. I think you have a dislike for `grep` :)

Reply:
OK; we don't need to make a change here for this to merge.

I wouldn't use `grep` on this line though; I don't think it's a good tool to teach in this context.

Or if we want to teach people how to use iproute2 a bit, I'd leave out the grep so they can see the detail in context (and, if they are a novice, they can look for the only element which is in CIDR notation).

We are helping people to _learn_ here rather than to achieve a task, so it's OK not to do every step for them.

---

Outside of a tutorial, you could use:
```shell
ip --json -f inet address show dev veth12ab34cd | jq -r '.[0] | .addr_info | .[] | ( [ .local, .prefixlen ] | join("/") )'
```
which gets you just the IP address ranges, in CIDR notation.

Reply:
Done. Thanks for the detailed explanation